apps/constants/base.py

```diff
@@ -510,6 +510,8 @@
     {'name': _('Level 9'), 'points': 3000000},
 ]
 
+REVIEW_LIMITED_DELAY_HOURS = 20
```

Comment:
I guess it could stay here then. Could you also add a comment above this summarizing what it's for?